source/api/discovery/0.5/index.md

```diff
@@ -82,127 +82,138 @@ The key words _MUST_, _MUST NOT_, _REQUIRED_, _SHALL_, _SHALL NOT_, _SHOULD_, _S
 {: #listing-resources-and-their-changes}
 
-There are three levels of conformance at which changes can be described. Level 0 is simply a list of the resources available.  Level 1 adds timestamps and ordering from earliest change to most recent, allowing a consuming application to work backwards through the list and stop processing once it encounters a change that it has already seen from a previous run. Level 2 adds information about the types of activities, enabling the explicit description of the creation and deletion of resources.
+There are three levels of conformance at which changes can be described. Level 0 is simply a list of the resources available. Level 1 adds timestamps and ordering from earliest change to most recent, allowing a consuming application to work backwards through the list and stop processing once it encounters a change that it has already seen from a previous run. Level 2 adds information about the types of activities, enabling the explicit description of the creation and deletion of resources.
 
 The subsections below describe first how to construct the description of the changes for each level, and then in the next section how to embed them into ordered lists.
 
 #### 2.1.1. Level 0: Resource References
 {: #level-0-basic-resource-list}
 
-The core information required to provide a minimally effective set of links to IIIF resources is just the URIs of those resources. However, with the addition of some additional JSON structure wrapped around those URIs, the document can also be on the path towards a robust set of information that clients can use to optimize their processing.
-
-Starting with the IIIF resource URIs, we add an "Update" Activity wrapper around them.  The order of the resources in the resulting list is unimportant, but each should only appear once. In terms of optimization, this approach provides no additional benefit over any other simpler list format, but is compatible with the following levels which introduce significant benefits.  This is the minimum level for any effective interoperability.
+The core information required to provide a minimally effective set of links to IIIF resources is the URIs of those resources. The order of the resources in the resulting list is unimportant, but each _SHOULD_ only appear once. This is the minimum level for any effective interoperability.
 
 If resources are deleted after being referred to in the resource list, the entire list should be republished without the reference to the deleted resource. Clients should also expect to encounter resource URIs that are out of date and no longer resolve to a IIIF Manifest or Collection.
 
 Example Level 0 Activity:
 
-```
-{
-  "type": "Update",
-  "object": {
-    "id": "https://example.org/iiif/1/manifest",
-    "type": "Manifest"
-  }
-}
+```json-doc
+[
+  {
+    "type": "Update",
+    "object": {
+      "id": "https://example.org/iiif/1/manifest",
+      "type": "Manifest"
+    }
+  },
+  // ...
+]
 ```
```

Comment:
The 2.1.x text refers to lists over and over; seems like the examples should be lists as well.

Reply:
-0. With an array, the example is an array with an update resource in it. I would then expect some property that has the array as its value ... which exists (`items`) but isn't the point of this part of the spec. Faint unease about this change.

Reply:
`orderedItems`, right? Anyway, curious what others think; it really stood out to me that the the whole section was about lists of changes but the examples weren't. Maybe others find it less jarring.

Reply:
Also -0.  It didn't strike me as odd, probably because the AS spec does it for examples of an individual, e.g. https://www.w3.org/TR/activitystreams-core/#activities

Reply:
I don't think the array works and note that the titles of these examples all say "Example Level 1 Activity" (singular) which doesn't match having a list.

I think the best thing would be to make a forward reference from the first example to  "See [pages of changes](https://iiif.io/api/discovery/0.3/#pages-of-changes) for how these activity objects are assembled into a list." or similar

Reply:
I removed the array wrappers in the latest push.

Reply:
If nobody else found it strange to be showing objects while talking about lists, I don't think we need a forward ref; it was probably just me.